frontend/public/components/dashboard/dashboards-page/overview-dashboard/queries.ts

```diff
@@ -53,6 +60,9 @@ const overviewQueries = {
   [OverviewQuery.STORAGE_UTILIZATION]:
     '(sum(node_filesystem_size_bytes) - sum(node_filesystem_free_bytes))',
   [OverviewQuery.STORAGE_TOTAL]: 'sum(node_filesystem_size_bytes)',
+  [OverviewQuery.POD_UTILIZATION]: 'count(kube_pod_info)',
+  [OverviewQuery.NETWORK_UTILIZATION]:
+    'sum(rate(container_network_receive_bytes_total{container="POD",pod!=""}[5m]) + rate(container_network_transmit_bytes_total{container="POD",pod!=""}[5m]))',
```

Comment:
```suggestion
    'sum(rate(container_network_receive_bytes_total{container="POD",pod!=""}[5m]) + sum(rate(container_network_transmit_bytes_total{container="POD",pod!=""}[5m]))',
```

nit: This is the first time I see network utilization measured as a sum of TX and RX. Usually, utilization is calculated as a percent of some maximum value, in case of network formula is `utilization % = (data bits x 100) / (bandwidth x interval)`.